### ISSUE TITLE
Do wxreactor.callLater usiing wxTimer

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7196]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#471|
|Type|enhancement|
|Created|2004-01-13 11:34:38Z|

<details><summary>Searchable metadata</summary>

```
trac-id__471 471
type__enhancement enhancement
reporter__itamarst itamarst
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1073993678000000 1073993678000000
changetime__1470654113831154 1470654113831154
version__ 
owner__ 
cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
See also patch on list.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub commented|
|-|-|

In changeset 800217c8d1aaa097d3ef0efcb8982467d5262c67
```
#!CommitTicketReference repository="" revision="800217c8d1aaa097d3ef0efcb8982467d5262c67"
Merge pull request #471 from twisted/8716-rodrigc-htb-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8716

Add test_htb and test_pcp to Python 3 tests
```
